cmd/server/main.go

```diff
@@ -3,10 +3,17 @@ package main
 import (
 	"log"
 
+	"github.com/OAuth2withJWT/identity-provider/internal/db"
 	"github.com/OAuth2withJWT/identity-provider/server"
 )
 
 func main() {
-	s := server.New()
+	DB, err := db.InitDB()
```

Comment:
It's okay to use the `db` variable name in this context, because we're only referring to the package when creating the connection so you won't have name clashes. But usually it's better to avoid it if possible.